packages/components/autocomplete/src/autocomplete.vue

```diff
@@ -194,6 +194,7 @@ const handleInput = (value: string) => {
   emit('input', value)
   emit(UPDATE_MODEL_EVENT, value)
   suggestionDisabled.value = false
+  activated.value = Boolean(value)
```

Comment:
Maybe no need to add code，just remove this line
https://github.com/element-plus/element-plus/blob/c098496f0a08c3b371eb3dc78ba9265e682e8afb/packages/components/autocomplete/src/autocomplete.vue#L218

Reply:
yeah, I know this line, but the suggestions should be closed after clear, this behavior is also the same in `element-ui2.x`

Reply:
Okay, I see.